src/Symfony/Component/Validator/Tests/Constraints/NotCompromisedPasswordValidatorTest.php

```diff
@@ -165,6 +165,30 @@ public function testInvalidPasswordCustomEndpoint()
             ->assertRaised();
     }
 
+    public function testEndpointWithInvalidValueInReturn()
+    {
+        $returnValue = implode(
+            "\r\n",
+            [
+                '36039744C253F9B2A4E90CBEDB02EBFB82D:5',
+                'This should not break the validator',
+                '3686792BBC66A72D40D928ED15621124CFE:7',
+                "36EEC709091B810AA240179A44317ED415C:2\r\n",
```

Comment:
```suggestion
                '36EEC709091B810AA240179A44317ED415C:2',
                '',
```
That should make it a bit more obvious which issue we're covering.